src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+
+type Props = {
+  onAddMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onAddMovie,
+}) => {
+  const [query, setQuery] = useState<string>('');
+  const [
+    searchError,
+    setSearchError,
+  ] = useState<ResponseError | undefined>(undefined);
+  const [
+    searchedMovie,
+    setSearchedMovie,
+  ] = useState<Movie | undefined>(undefined);
```

Comment:
```suggestion
  ] = useState<Movie | null>(null);
```
use `null` in such cases